src/store/users/saga.ts

```diff
@@ -0,0 +1,32 @@
+import { takeLatest, put, call } from 'redux-saga/effects';
+import { SagaActionTypes } from '.';
+import { channelIdPrefix } from '../channels-list/saga';
+import { receive } from '../channels';
+
+import { fetchUsersByChannelId } from './api';
+
+export interface Payload {
+  channelId: string;
+}
+
+export function* fetch(action) {
+  const { channelId } = action.payload;
+  const channelPrefix: string = channelIdPrefix + channelId;
+
+  const users = yield call(fetchUsersByChannelId, channelPrefix);
+  const formatUsers = users.map(({ userId: id, ...rest }) => ({
+    id,
+    ...rest,
+  }));
+
+  yield put(
+    receive({
+      id: channelId,
+      users: formatUsers,
+    })
+  );
+}
```

Comment:
i don't think this belongs in the user saga. this method is too generic, since this feature isn't about fetching users, it's about fetching channel members. the saga should be on the channel, not the user. the problem here is that in the future, once we need to fetch users for some other purpose, we need to change this method. let's pull this into the channel saga, and name it appropriately for what it's doing. eg. `loadUsers` or something.